Docs: fix missing values use-case in tutorial for `PandasDataset`

cneely33 opened the pull request:
The reindexing of the grouped dataframes is being performed on the original dataframe. It should be preformed on the dataframe where rows were removed, df_missing_val.


The freq of pd.date_range on line 127 is also set to "1D" instead of "1H" as in the original dataframe.

*Description of changes: Update pandasdataframes.md.template

cneely33 committed:
Update pandasdataframes.md.template

The reindexing of the grouped dataframes is being performed on the original dataframe. It should be preformed on the dataframe where rows were removed, df_missing_val.

melopeo commented:
Hello @cneely33 ,

thanks for this PR. Your observations seem to be correct. To go ahead with your PR, could you please as well update the frequency as you have pointed out, i.e. from "1D" to "1H"?

Thanks!

melopeo commented on docs/tutorials/data_manipulation/pandasdataframes.md.template:
```diff
@@ -123,7 +123,7 @@ from gluonts.dataset.pandas import PandasDataset
 dfs_dict = {}
-for item_id, gdf in df.groupby("item_id"):
+for item_id, gdf in df_missing_val.groupby("item_id"):
     new_index = pd.date_range(gdf.index[0], end=max_end, freq="1D")
```
As you have observed, can you please updated `freq="1D"` to `freq="1H"`?

cneely33 replied:
Done. Thank you.

melopeo reviewed (changes requested):
As you have suggested, can you please updated `freq="1D"` to `freq="1H"`?

cneely33 committed:
Update pandasdataframes.md.template

Correcting error by changing frequency of new dataframe index from Daily to Hourly

melopeo reviewed (approved):
LGTM

lostella commented:
Thanks @cneely33! I think we can merge this, and see about style check failures separately
